wplms-coauthor-plus.php

```diff
@@ -44,19 +44,24 @@ function wplms_coauthor_plus_instructor($instructor, $id,$r = null){
       foreach($coauthors as $k=>$inst){
         $instructor_id = $inst->ID;
         $displayname = bp_core_get_user_displayname($instructor_id);
-        if(function_exists('vibe_get_option'))
+        if(function_exists('vibe_get_option')) {
           $field = vibe_get_option('instructor_field');
+          if(!isset($field) || $field =='') $field='Speciality';
```

Comment:
The closing bracket should close here so that we have a default value for the $field, when it is used in the bp_get_profile_field_data function.